iceoryx2-pal/posix/build.rs

```diff
@@ -17,6 +17,9 @@ use bindgen::*;
 use std::env;
 use std::path::PathBuf;
 
+// ICEORYX2_CLANG_ARG is used to specify the clang_arg for bindgen.
```

Comment:
To be consistent with the renaming it should be
```suggestion
// IOX2_CLANG_ARG is used to specify the clang_arg for bindgen.
```

Looking at the corresponding issue, is it really necessary to introduce a new env variable or could the same be achieved with the `BINDGEN_EXTRA_CLANG_ARGS` env variable, which is mentioned in the error message. Please check https://github.com/rust-lang/rust-bindgen?tab=readme-ov-file#environment-variables

If it does not work, it might make sense to call the env variable `IOX2_BINDGEN_EXTRA_INCLUDE_PATH` to be more specific.

Reply:
oh my bad, I export `export BINDGEN_EXTRA_CLANG_ARGS=-I/usr/lib/gcc/x86_64-linux-gnu/9/include` and it works for the bindgen issue. sorry that i have missed the bindgen support.

thanks for your suggestion, and let me close my PR as it doesn't make sense now.